### DISCUSSION TITLE
Budibase Cloud - Where are my apps?

### DISCUSSION THREAD

**Discussion by alexclusively** [Budibase/budibase#3212] [Help]
I have tried creating some apps from the "Get started quickly" section of the apps builder but every time I did it would kick me back to the beginning with no indication as to where my app went. Now, after a few attempts, it is saying I have reached my app quota. Where are my apps and how do I access them?

**Comment by mjashanks** [comment]
Hi There.

Doesn't sound good. This works for me ok. Could you please support@budibase.com your Budibase domain e.g. `mydomain.budibase.app`, so I can investigate.

Thanks!

**Reply by luke2727io** [reply]
Hi,

Have the same problem with my Budibase, tried emailing but bonces back saying the email support@budibase.com doesn't exist

**Reply by mjashanks** [reply]
Sorry, email me... address in my profile.

**Reply by mjashanks** [reply]
We had an issue whereby using a `_` in your domain name caused this issue. Please delete your account and signup again, without the underscore.

Thanks for reporting this!

**Reply by mjashanks** [reply]
@luke2727io and @alexclusively - we have now fixed this issue. Thank you for reporting it.

we had to strip the `_` from your domain. So:

If your domain was `jane_smith.budibase.app`, you should now be able to use it under `janesmith.budibase.app`. Sorry about this :)
